mswindows/osgeo4w/package.sh

```diff
@@ -203,6 +208,8 @@ mv $OSGEO4W_ROOT_MSYS/apps/grass/grass$POSTFIX/include/grass/config.h \
    $OSGEO4W_ROOT_MSYS/apps/grass/grass$POSTFIX/include/grass/config.h.mingw
 cp mswindows/osgeo4w/config.h.switch $OSGEO4W_ROOT_MSYS/apps/grass/grass$POSTFIX/include/grass/config.h
 cp mswindows/osgeo4w/config.h.vc $OSGEO4W_ROOT_MSYS/apps/grass/grass$POSTFIX/include/grass
+mv $OSGEO4W_ROOT_MSYS/apps/grass/grass$POSTFIX/etc/grass.py $OSGEO4W_ROOT_MSYS/apps/grass/grass$POSTFIX/etc/grass${POSTFIX}.py
```

Comment:
What about fixing https://github.com/OSGeo/grass/blob/main/mswindows/osgeo4w/grass.bat.tmpl#L18 instead of introducing `mv` statement?

Reply:
I tried to modify `grass.bat` locally but it end up with error
![VirtualBox_Windows 10 (ArcGIS Pro)_05_10_2021_14_09_54](https://user-images.githubusercontent.com/5683186/136020872-c10082dd-db43-45a4-8f90-ebc562398f2a.png)

So the move statement approach seems to be correct.